service/history/conflictResolver.go

```diff
@@ -85,9 +91,9 @@ func (r *conflictResolverImpl) reset(requestID string, replayEventID int64, star
 		// NextEventID could be in the middle of the batch.  Trim the history events to not have more events then what
 		// need to be applied
 		if batchSize > eventsToApply {
-			history.Events = history.Events[0:eventsToApply]
+			r.logError("Conflict resolution err mis-allignment history.", ErrCorruptedHistory)
```

Comment:
Probably this error is now redundant.

Reply:
ok removed

Reply:
actually, i would like to keep this one, since when using ELK to go through error logs in history service, this can be helpful.